scripts/legislation/africa/uganda_english.sh

```diff
@@ -0,0 +1,287 @@
+    #Article 35
+    amend_error_in_article 35 'Limits on the use of lakes and rivers. ' '' | \
+    amend_error_in_article 35 '36-' '36:' | \
+    #Article 37
+    amend_error_in_article 37 'oris' 'or is' | \
+    #Article 39
+    amend_error_in_article 39 'degradation if' 'degradation if: ' | \
+    #Article 42
+    amend_error_in_article 42 '\]-' ']:' | \
+    #Article 43
+    amend_error_in_article 43 '\]i' '\] i' | \
+    amend_error_in_article 44 'exsitu' 'ex-situ' | \
+    #Article 46
+    amend_error_in_article 46 'thelead' 'the lead' | \
+    amend_error_in_article 46 'section \[3\]' 'section 3.' | \
+    amend_error_in_article 46 'whall' 'shall' | \
+    #Article 47
+    amend_error_in_article 47 'section \[3\]' 'section 3.' | \
+    amend_error_in_article 47 'byindividual landusers' 'by individual land users' | \
+    #Article 51
+    amend_error_in_article 51 '\(c' '[c]' | \
+    #Article 55
+    amend_error_in_article 55 'fallingunder' 'falling under' | \
+    amend_error_in_article 55 '\[54\]' '54.' | \
+    amend_error_in_article 55 'an offence ' 'an offence. ' | \
+    #Article 56
+    amend_error_in_article 56 'measures on' 'measures on: ' | \
+    #Article 57
+    amend_error_in_article 57 'discharge by' 'discharge by: ' | \
+    #Article 59
+    amend_error_in_article 59 'detemine' 'determine' | \
+    amend_error_in_article 59 'Comittee' 'Committee' | \
+    #Article 61
+    amend_error_in_article 61 '\]-' ']:' | \
+    #Article 64
+    amend_error_in_article 64 '\[4\]' '. [4]' | \
+    amend_error_in_article 64 'further information .' 'further information.' | \
+    #Article 65
+    amend_error_in_article 65 'orin' 'or in' | \
+    #Article 68
+    amend_error_in_article 68 '\]levying' '] levying' | \
+    #Article 73
+    amend_error_in_article 73 '\[77\]' '77.' | \
+    amend_error_in_article 73 'so as to' 'so as to: ' | \
+    #Article 75
+    amend_error_in_article 75 '. easement.' '.' | \
+    amend_error_in_article 75 'Enforcement of environmental ' '' | \
+    #Article 76
+    amend_error_in_article 76 ' Cap.' '' | \
+    #Article 79
+    amend_error_in_article 79 '\[87\]' '87.' | \
+    #Article 81
+    amend_error_in_article 81 'without warrant' 'without warrant: ' | \
+    #Article 88
+    amend_error_in_article 88 'Integration of environmental education i the school curriculum.' '' | \
+    #Article 90
+    amend_error_in_article 90 '91' '\n\n(91)' | \
+    #Article 93
+    amend_error_in_article 93 'appoint by him' 'appointed by him' | \
+    #Article 96
+    amend_error_in_article 96 'requiremnt' 'requirement' | \
+    #Article 106
+    amend_error_in_article 106 'Porfeiture, cancellation, community service and other orders. ' '' | \
+    #Article 108
+    amend_error_in_article 108 'bye-laws' 'by-laws' | \
+    amend_error_in_article 108 'cenvenient' 'convenient' | \
+    #Article 109
+    amend_error_in_article 109 'Existing law.*' ''
+}
+
+function remove_and_reinsert_article_titles {
```

Comment:
Now I see how you are using the `remove_and_reinsert_article_title` function. I believe that the use case justifies the function's existence. My only request is that you replace the last line of the `remove_and_reinsert_article_title` function, i.e.

```
sed -E "s/\(${article_number}\)/${article_title}\n(${article_number})/"
```

with

```
sed -E "s/^\(${article_number}\)/${article_title}\n(${article_number})/"
```

That modification will ensure that `sed` only matches lines beginning with the provided article number. Please include the change in the next revision that you post to this branch.

Reply:
Actually, I need to request a second modification. When deleting the article text (the second-to-last line of the `remove_and_reinsert_article_title` function, please ensure that the function only deletes the title in the target article.

Reply:
I'm not sure what you mean by this. Isn't `sed -E "s/${article_title}//"` currently operating to delete only the title of the target article?

Reply:
The problem is that `sed -E "s/${article title}//"` will delete `${article_title}` wherever it occurs in the legislation. If an article other than the target article contains the `${article_title}` text, that text will also be deleted.

Reply:
That is the intention of the function. The column formatting of the article titles can result in the article titles being grouped up in unrelated articles. Thus, the article titles can show up unwanted in unrelated articles. 

For example this shows up at the end up article 12 of the Malawi legislation "Disclosure of interest Allowances of members of the Council Technical Committee on the Environment". This is actually the article titles for 3 different articles that have been grouped together and inserted into article 12 due to their column formatting. 

The capitalization in the article titles should limit the possibility of any false positives.

Reply:
I see. Okay, that's fine then. Please push a new revision that addresses my other comments above. Thanks.